Signum.React.Extensions/Chart/GoogleMapScripts/Heatmap.tsx

```diff
@@ -19,6 +19,8 @@ export default class HeatmapChart extends React.Component<ChartClient.ChartCompo
   divElement?: HTMLDivElement | null;
 
   render() {
+    this.divElement && this.drawChart(this.props);
+

```

Comment:
This is a side effect in render function. not sure if/how should be translated to hooks. We have to check together